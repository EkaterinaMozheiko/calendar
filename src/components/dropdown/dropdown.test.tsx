import { shallow } from 'enzyme';
import React from 'react';

import Dropdown, { DropdownProps } from './dropdown';

describe('<Dropdown />', () => {
  let props: DropdownProps;
  const currencies = [
    { currencyName: 'UAE Dirham', id: 'AED' },
    { currencyName: 'Netherlands Antillean Gulden', currencySymbol: 'Æ’', id: 'ANG' }
  ]
  beforeEach(() => {
    props = {
      theme: null,
      classes: null,
      placeholder: 'From',
      currencies
    }
  });

  it('Dropdown should match snapshot', () => {
    const appComponent = shallow(<Dropdown {...props} />);
    expect(appComponent).toMatchSnapshot();
  })

  it('render currencies', () => {
    const wrapper = shallow(
      <Dropdown {...props} />
    );
    expect(wrapper.prop('currencies')).toEqual(currencies);
  });

});
