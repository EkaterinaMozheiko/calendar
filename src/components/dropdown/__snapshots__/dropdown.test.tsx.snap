// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dropdown /> Dropdown should match snapshot 1`] = `ShallowWrapper {}`;
